### PR TITLE
Sandra Mauro hw01

### PR DESCRIPTION
**Pull request by sandramauro**
I found the assignment fairly straightforward after watching the video. I have used r markdown locally before, but I had a bit of trouble adjusting to using the server version because I did not know where files were saved. I have never used github before, but after watching the video I think I figured out the basics. I had some trouble adding an image to my README. I found the r markdown cheatsheet very useful for this assignment.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
+![](images/IMG_9165.png)
+
+**Sandra Mauro**
+
+My name is Sandra Mauro and I am a first-year student at UChicago.
+I am currently in the SISRM program.
+
+
+These are my summer plans:
+ + Take Computing for the Social Sciences
```
Comment (wanitchayap) [inline README.md:14]: ❤️

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+ + Conduct research with Professor Casey Mulligan
+ + Take an online python course
```
Comment (wanitchayap) [inline README.md:16]: Learning both R and Python at the same time could be helpful! I think it helped me understand both languages better and faster. But things can get confusing sometimes too 😆

**Review by wanitchayap — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Improvement |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |   X        |              |            |
| **Coding strategy**         |      X     |              |            |
| **Presentation**            |           |       X       |            |
| **Achievement, creativity** |     X      |              |            |
| **Reproducibility**         |     X      |              |            |

Remarks:
* Great job!
* Frequent commits + detail messages 👍

**Comment by jackocyber** [comment]
I liked the straightforward nature to you biography and the many commits displayed that you have a good understanding of the assignment. I also liked they way that you reflected through the github/hw workflow. I think you could have added some bold or some other items to present your bio to spice it up, but other than that I think it was great!

**Pull request closed without merging**